.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,32 @@
+
+name: Publish package to PyPi
+
+on:
+  release:
```

Comment:
I'm not a fan of requiring GitHub releases to get PyPI releases, but this is the current pattern, so whatever.

Reply:
Yeah 😛 Out of curiosity, what do you prefer? Automatically push to pypi whenever the version number changes?

Reply:
(btw, this file is getting deleted from the PR, since there is already a `pypi-release.yml` in this repository)